Assets/Scripts/Shop/ShowStoreOnCollide.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ShowStoreOnCollide : MonoBehaviour
```

Comment:
As noted in discord, we should have this accessible to the user via a keyboard button, always